module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php

```diff
@@ -409,6 +409,7 @@ public function getDefaultDescriptionSpecs()
         $spec->setLine('Access', 'getAccessRestrictions');
         $spec->setLine('Finding Aid', 'getFindingAids');
         $spec->setLine('Publication_Place', 'getHierarchicalPlaceNames');
+        $spec->setLine('Source', 'getSource');
```

Comment:
If we rename the Source facet to Database, would do the same here.  The method though is an existing one getSource in Record Driver.  https://vufind.org/wiki/development:architecture:record_driver_method_master_list

Reply:
Per another thread above, not renaming this.